pkg/parcacol/querier.go

```diff
@@ -129,8 +135,52 @@ func (q *Querier) Values(
 ) ([]string, error) {
 	vals := []string{}
 
+	// TODO(ivo) as we do not know if the labelName is a parca or pprof label
```

Comment:
since we ensure on insertion that there is no conflict, we can do a single query that contains both columns in the distinct operation, we'll never get a value in both columns

Reply:
perfect 😄